### PR TITLE
hugo-server: add page

### PR DESCRIPTION
**Pull request by nelsonfigueroa**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).

### CODE REVIEW

File: pages/common/hugo-server.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+- Build and serve a site:
+
+`hugo server`
+
+- Build and serve a site on a specified port number:
+
+`hugo server --port {{port_number}}`
+
+- Build and serve a site while minifying supported output formats (HTML, XML, etc):
```
Comment (spageektti) [inline pages/common/hugo-server.md:14]:
```suggestion
- Build and serve a site while minifying supported output formats (HTML, XML, etc.):
```

**Review by pixelcmtd — APPROVED** [review]
LGTM 😄

File: pages/common/hugo-server.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+`hugo server`
+
+- Build and serve a site on a specified port number:
+
+`hugo server --port {{port_number}}`
+
+- Build and serve a site while minifying supported output formats (HTML, XML, etc.):
+
+`hugo server --minify`
```
Comment (spageektti) [inline pages/common/hugo-server.md:16]:
```suggestion
`hugo server --minify`

- Display help:

`hugo server --help`
```

**Review by spageektti — APPROVED** [review]
Thanks. LGTM.

**Pull request merged**